Jasmine/LevelDesigner/ViewControllers/LevelDesignerViewController.swift

```diff
@@ -35,17 +41,19 @@ class LevelDesignerViewController: UIViewController {
 
     /// Feeds in the appropriate data for the use of seguing into this view.
     ///
-    /// - Parameter viewModel: the level designer view model required to use this view
-    func segueWith(_ viewModel: LevelDesignerViewModel) {
+    /// - Parameters:
+    ///   - viewModel: the level designer view model required to use this view
+    ///   - onSaveCallBack: Callback to be executed after saving
+    func segueWith(_ viewModel: LevelDesignerViewModel, onSaveCallBack: @escaping () -> Void) {
```

Comment:
Oh u meant to let the designer VC to store a LevelSelectorViewControllerDelegate? I think just registering callback is simpler. I did that for phrase explorer VC to designer as well.